### DISCUSSION TITLE
jakarta.mail.AuthenticationFailedException with valid credentials

### DISCUSSION THREAD

**Discussion by Tasza33** [eclipse-ee4j/mail#727] [Q&A] [answered]
Hi community. I'm trying to send email without SSL through that [repo](https://github.com/qa-guru/allure-notifications), in which your methods are used. So my config there looks like:

```
\"mail\": {
        \"host\": \"mail.com\",
        \"port\": \"25\",
        \"username\": \"svc..@mail.com\",
        \"password\": \"${CI_EMAIL_PASSWORD}\",
        \"securityProtocol\": \"null\",
        \"from\": \"svc..@mail.com\",
        \"recipient\": \"n...@mail.com\"
  } 
```

My credentials are valid, that was tested. Maybe you know what's going wrong?
Full log [java.log](https://github.com/qa-guru/allure-notifications/files/14873304/java.log)

Used image [dquintela/openjdk-8-jdk-alpine](https://hub.docker.com/layers/dquintela/openjdk-8-jdk-alpine/latest/images/sha256-612698b405dbd06bfc449d12677aa10ce4625425ed2a6ccfab401ee2d6ac8d38)

**Comment by jmehrens** [comment, marked as answer]
Most likely it is this issue: [javax.mail.AuthenticationFailedException: 535 5.7.3 Authentication unsuccessful](https://stackoverflow.com/questions/33243450/javax-mail-authenticationfailedexception-535-5-7-3-authentication-unsuccessful) which is issues with exchange permissions or unable to relay messages from the account you are using.
